tests/v1/lookup_client/test_lmcache_lookup_client.py

```diff
@@ -212,7 +213,8 @@ def test_basic_lookup_communication(self, lmcache_engine):
 
                 # Test clear lookup status
                 client.clear_lookup_status(lookup_id)
-                assert client.lookup_cache(lookup_id) is None
+                # After clearing, lookup_cache returns -1 (not found)
+                assert client.lookup_cache(lookup_id) == -1
```

Comment:
This has been fixed in another PR and merged, would you like to rebase your branch? @mustafayildirim